open_humans/templates/base.html

```diff
@@ -109,14 +109,17 @@
 
             <li class="{% active 'login' %}">
               <p class="navbar-btn">
-                <a href="{% url 'account_login' %}?next={% get_next_url %}" class="btn btn-default login-link">Log in</a>
+                <a onClick="return storeRedirect();"
```

Comment:
not going to work when JS is disabled

Reply:
this looks like a reason to be passing a `next` parameter to Login and Signup views – and then in those form processing events, set the session `next_url` based on the presence of a hidden `next` parameter in the form, but only _if it's not already set_